libs/sdk-ui-charts/src/charts/headline/internal/HeadlinePagination.tsx

```diff
@@ -0,0 +1,32 @@
+// (C) 2021 GoodData Corporation
+import React from "react";
+import cx from "classnames";
+
+interface IHeadlinePaginationProps {
+    item: number;
+    showPrevItem: () => void;
+    showNextItem: () => void;
+}
+
+export const HeadlinePagination: React.FC<IHeadlinePaginationProps> = ({
+    item,
+    showPrevItem,
+    showNextItem,
+}) => {
+    const getClassNames = () => cx("gd-button-link", "gd-button-icon-only", "pagination");
+
+    return (
+        <div className="headline-pagination">
+            <button
+                className={`${getClassNames()} first-item icon-chevron-left`}
+                onClick={showPrevItem}
+                disabled={item === 1}
+            />
```

Comment:
should not this component know also the items array, and its length? Then index=0 means first and index=length-1 means last and it can be more universal. But if it is too much work, let keep hardcoded just for to items of secondary and ternary value